### ISSUE TITLE
Enhancing Permission Security in Tron

### ISSUE THREAD

**Issue by zonanaza** [tronprotocol/tips#676]
Context: The increasing number of phishing attacks through cloned websites offering fake airdrops or rewards is compromising the security of Tron accounts. Users unknowingly delegate full control of their accounts to malicious actors by signing transactions, losing control despite still holding the private keys.

Problem: Currently, when the Owner permission is delegated to another account, the original private key holder cannot recover control. This issue not only impacts individual users targeted by scams but also affects businesses using multisignature systems. If an employee with critical permissions leaves the company, the original account owner cannot revoke those permissions, leaving the account vulnerable.

Practical Examples:

Phishing Attacks: A user signs a transaction to claim an airdrop but unknowingly delegates the Owner permission to a fraudulent account, losing access to their account.
Business Management: In a multisignature setup, if an employee leaves and has delegated permissions, the account remains exposed if those permissions cannot be revoked.
Proposed Improvement: Introduce a feature allowing the private key holder to reverse any permission updates, including the Owner permission. This would ensure that the legitimate account owner retains ultimate control over the account.

Justification:

User Security: Protect users from phishing scams and frauds that could result in the loss of control over their accounts.
Business Flexibility: Enable businesses to manage multisignature systems more efficiently, maintaining control over permissions even in case of personnel changes.
Increased Trust in Tron: Strengthening control over permissions will enhance the platform’s security and boost user confidence, making Tron more resilient against attacks.
Conclusion: The private key holder must have the ultimate authority to revoke any permission, ensuring they always retain full control of the account. This improvement would protect users from phishing attacks and enhance overall security for assets managed in Tron, offering more effective and secure account management.

**Comment by zeusoo001** [comment]
@zonanaza There are pros and cons. If the private key holder has the final power to revoke any permissions, if the private key is stolen, the assets will be lost.

**Comment by KrisdiaPaul** [comment]
@zonanaza I completely agree with your point about the problems caused by losing permissions, and I am also thinking about it. Strengthening the private key holder was once an idea that I thought of, but I quickly rejected it. It just involves more problems and none of block chain multi-signature method allow this.

**Comment by halibobo1205** [comment]
Add permission for a limited period, e.g. 1 year by default?

**Comment by halibobo1205** [comment]
@zonanaza Is this still active? If not, it'll be closed soon.

**Comment by rmcharts** [comment]
Can someone confirm if this TRX wallet   TMDxvJbsBTBJcyxfdrdnLYJKdhtgacHadd   is set up correctly? can help me?

**Comment by halibobo1205** [comment]
@rmcharts, can you be more specific about setting up correctly?

**Comment by ferdinand026** [comment]
> Can someone confirm if this TRX wallet TMDxvJbsBTBJcyxfdrdnLYJKdhtgacHadd is set up correctly? can help me?

@rmcharts How did you get this address? Do you have the privatekey of this address? If this address is created by yourself offline and you ensure that the private key information of this account has not been leaked, you can use this address.  But this address has not been activated, so it cannot be searched in the [tronscan](https://tronscan.org/#/address/TMDxvJbsBTBJcyxfdrdnLYJKdhtgacHadd). If you want to activate this account, just [transfer some TRX into it](https://developers.tron.network/docs/account#account-activation). 

![Image](https://github.com/user-attachments/assets/00b0f661-49a9-4809-8151-39693d775951)

**Comment by notblaze11** [comment]
Hello. My wallet owner and active permission was recently stolen. No idea why the person who did this have not taken the money yet. And the address that multi sig my wallet is inactive (never been activated). Please how can I reverse this as I can see my money in wallet but can’t send out only usdt and trx. Thank you
